tasks/build_tags.py

```diff
@@ -92,6 +94,7 @@
 # AGENT_HEROKU_TAGS lists the tags for Heroku agent build
 AGENT_HEROKU_TAGS = AGENT_TAGS.difference(
     {
+        "bundle_installer",
```

Comment:
Is the heroku agent using the installer?

Reply:
It's not and I'm not adding it in Heroku (see the package size increases).
Difference gives you the XOR of the two arrays, honestly it is super duper confusing to use that.

Reply:
Ahh you are right 🤦 

We could change the `AGENT_HEROKU_TAGS` to be a list instead of using `difference`. Similar to what we did here https://github.com/DataDog/datadog-agent/pull/33528

Is outside of the scope of your change, so I can understand if you prefer if we do it on another PR